### PR TITLE
Regenerate with latest gir

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: gtk4/src/expression.rs [hunk 1]
```diff
@@ -23,6 +23,14 @@ pub unsafe trait IsExpression:
 }
 
 impl Expression {
+    pub fn upcast(self) -> Self {
```
Comment (bilelmoussaoui) [inline gtk4/src/expression.rs:26]: Really not happy with this :(

Comment (sdroege) [reply gtk4/src/expression.rs:26]: Well, theoretically you should have a trait that implements this on all expression types (including `Expression`), but until then having it as a method on `Expression` (like on all other expression types) would be symmetric at least.

Comment (bilelmoussaoui) [reply gtk4/src/expression.rs:26]: We already have those methods on a trait actually, but it is only implemented for the sub-types. I will see what can be done here

Comment (bilelmoussaoui) [reply gtk4/src/expression.rs:26]: The problem with going through a trait is auto-generated bindings won't include it :( so we really need to start importing `use glib::prelude::*;` & `use $crate::prelude::*;` instead from gir side.

Comment (sdroege) [reply gtk4/src/expression.rs:26]: `use $crate::prelude::*` should already include the `glib` prelude or not?

Comment (bilelmoussaoui) [reply gtk4/src/expression.rs:26]: it should indeed yes but a task for another time

**Pull request merged**